sdk/cli/README.md

```diff
@@ -0,0 +1,10 @@
+# Default CLI
+
+This directory contains the default CLI build, used by the majority of services who don't add custom commands to their service CLIs. The default CLI builds are uploaded with every SDK release as of early 2018.
```

Comment:
Could we reference a SHA here?

Reply:
Updated the package tooling to support template params like `{{sha256:filename@http://.../SHA256SUMS}}`. Now packages consuming default CLIs will be able to just specify the manifest URL for the SDK release they're using and the sha256sum will be fetched from there automatically.